independent-publisher-connectors/Shopify/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1808 @@
+                        "required": false,
+                        "type": "string",
+                        "description": "Show orders attributed to a certain app, specified by the app ID.",
+                        "x-ms-summary": "Attribution App ID"
+                    },
+                    {
+                        "name": "status",
+                        "in": "query",
+                        "required": false,
+                        "type": "string",
+                        "description": "Filter orders by their status.",
+                        "x-ms-summary": "Order status"
+                    },
+                    {
+                        "name": "financial_status",
+                        "in": "query",
+                        "required": false,
+                        "type": "string",
+                        "description": "Filter orders by their financial status.",
+                        "x-ms-summary": "Financial status"
+                    },
+                    {
+                        "name": "fulfillment_status",
+                        "in": "query",
+                        "required": false,
+                        "type": "string",
+                        "description": "Filter orders by their fulfillment status.",
+                        "x-ms-summary": "Fulfillment status"
+                    },
+                    {
+                        "name": "fields",
+                        "in": "query",
+                        "required": false,
+                        "type": "string",
+                        "description": "Retrieve only certain fields, specified by a comma-separated list of fields.",
+                        "x-ms-summary": "Fields"
+                    },
+                    {
+                        "name": "order",
+                        "in": "query",
+                        "required": false,
+                        "type": "string",
+                        "description": "Field and direction to sort orders. To specify the direction, add ' asc' or ' desc' to the end of the query parameter. E.g. 'order=updated_at asc'.",
+                        "x-ms-summary": "Sequence order of records"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "OK",
+                        "schema": {
+                            "type": "object",
+                            "properties": {
+                                "orders": {
+                                    "type": "array",
+                                    "items": {
+                                        "$ref": "#/definitions/order"
+                                    }
+                                }
+                            }
+                        }
+                    },
+                    "default": {
+                        "description": "default",
+                        "schema": {}

```

Comment:
@raybennett-msft schema is not allowed within the default response. please remove it. please fix all default responses in the swagger file. please see [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/coding-standards#responses).